.taskcluster.yml

```diff
@@ -69,7 +69,7 @@ tasks:
                 source: ${event.repository.url}
               payload:
                 image:
-                    harjgam/web-platform-tests:0.33
+                    filoop/web-platform-tests:0.34
```

Comment:
This now conflicts with master

Reply:
I see that both `harjgam/web-platform-tests:0.33` and `harjgam/web-platform-tests:0.34` appear in master now. Is that intentional, need the decision task and the rest not use the same image?